src/changes.ts

```diff
@@ -82,8 +79,8 @@ export async function prepareForDiff(): Promise<FileVersions[]> {
   if (files.length === 0)
     return [];
 
-  const baseRepoUrl = base.repo.git_url;
-  const headRepoUrl = head.repo?.git_url;
+  const baseRepoUrl = base.repo.clone_url;
+  const headRepoUrl = head.repo?.clone_url;
```

Comment:
I am not sure why I used the `git_url` initially but `clone_url` seems to have the proper https value 

https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28